style.css

```diff
@@ -1,87 +0,0 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
-
-html {
-  background: #191765;
-  display: block;
-}
-
```

Comment:
- [ ] [OPTIONAL] Kindly let's add the property `overflow-x: hidden;` to the body tag to remove the horizontal overflow which may cause a bad user experience 👍